perception/occupancy_grid_map_outlier_filter/include/occupancy_grid_map_outlier_filter/occupancy_grid_map_outlier_filter_nodelet.hpp

```diff
@@ -46,18 +46,17 @@ using geometry_msgs::msg::Pose;
 using nav_msgs::msg::OccupancyGrid;
 using sensor_msgs::msg::PointCloud2;
 using std_msgs::msg::Header;
-using PclPointCloud = pcl::PointCloud<pcl::PointXYZ>;
+using PclPointCloud = pcl::PointCloud<pcl::PointXYZI>;
```

Comment:
Is it possible to support XYZ so that it works with XYZ and XYZI?
If it doesn't work without XYZI, it will be difficult to use it with other sensors.

Reply:
@yukkysaito do you have other concerns related with this?